packages/mobile/README.md

```diff
@@ -193,6 +193,13 @@ The service that route SMS messages to the app needs to be configured to [append
 We're using [GraphQL Code Generator][graphql code generator] to properly type
 GraphQL queries. If you make a change to a query, run `yarn build:gen-graphql-types` to update the typings in the `typings` directory.
 
+## Generating SMS_RETRIEVER_HASH_CODE
```

Comment:
Kind of duplicates the info in the Configuring the SMS Retriever section just above.